sdk/lib/VirtIO/osdep.h

```diff
@@ -0,0 +1,46 @@
+//////////////////////////////////////////////////////////////////////////////////////////
+// Copyright (c) 2007  Qumranet All Rights Reserved
+//
+// Module Name:
+// osdep.h
+//
+// Abstract:
+// Windows OS dependent definitions of data types
+//
+// Author:
+// Yan Vugenfirer  - February 2007.
+//
+//////////////////////////////////////////////////////////////////////////////////////////
+
+#pragma once
+
+#include <ntddk.h>
+
+#ifdef __REACTOS__
+#ifdef __GNUC__
+#undef FORCEINLINE
+#define FORCEINLINE __attribute__((__always_inline__))
+#endif
+#endif
+
+#ifndef __REACTOS__
+#define ENOSPC 1
+#endif
+
+#if !defined(__cplusplus) && !defined(bool)
+// Important note: in MSFT C++ bool length is 1 bytes
+// C++ does not define length of bool
+// inconsistent definition of 'bool' may create compatibility problems
+#define bool u8
+#define false FALSE
+#define true TRUE
+#endif
+
+#ifdef __REACTOS__
+#ifdef __GNUC__
+#undef inline
+#define inline __attribute__((__always_inline__))
```

Comment:
This looks a little strange for me. It doesn't compile without this, right?

Reply:
No, it doesn't. It can compile without this, I tested. I just copy it from #1561

Reply:
Remove it then. The diff should be minimal, and this should not affect performance

Reply:
Yes, I removed